Assignment1/ho-taek_BESOPTAssignment/app/src/main/java/com/htk/sopt1/HomeActivity.kt

```diff
@@ -0,0 +1,46 @@
+package com.htk.sopt1
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import com.htk.sopt1.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding :ActivityHomeBinding
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.etHomename.setText("hotaek Kwak")
+        binding.tvHomeid.text = intent.getStringExtra("id")
+        binding.etHomeintroduce.setText("안드 사랑해 보고싶다. 안드 사랑 해보고싶다.")
```

Comment:
```suggestion
        with(binding) {
            etHomename.text = "hotaek Kwak"
            tvHomeid.text = intent.getStringExtra("id")
            etHomeintroduce.text = "안드 사랑해 보고싶다. 안드 사랑 해보고싶다."
        }
```

이렇게도 작성이 가능합니다. 코틀린에서 Scope 함수 활용해보면 Binding 코드를 더 깔끔하게 작성할 수 있으니 공부하시는 걸 추천드려요

Reply:
다른 분들 코드를 볼 때 let, apply, with, also를 보며 저게 도대체 뭘까??했었는데 Scope 함수라는 거군요!! 공부해서 코드 작성할때 활용하도록 하겠습니다